editor.html

```diff
@@ -6,6 +6,15 @@
     <script type="text/javascript" src="https://formit3d.github.io/FormItExamplePlugins/SharedPluginFiles/FormIt.js"></script>
     <script type="text/javascript" src="https://formit3d.github.io/FormItExamplePlugins/SharedPluginFiles/FormItInterface.js"></script>
     <script type="text/javascript" src="https://formit3d.github.io/FormItExamplePlugins/SharedPluginFiles/PluginUtils.js"></script>
+    
+    <!-- Monaco Support -->
+    <script>var require = { paths: { 'vs': 'https://cdnjs.cloudflare.com/ajax/libs/monaco-editor/0.27.0/min/vs' } };</script>
```

Comment:
This is a bit hacky, but I guess I'm not opposed to it since it's simpler. Something to keep in mind if we bring in another lib that needs it. Could just use require: https://requirejs.org/

Reply:
I thought this too but only put it there because of the example I used to build this. I'll see if I can take it out